decidim-system/app/forms/decidim/system/update_organization_form.rb

```diff
@@ -4,108 +4,36 @@
 
 module Decidim
   module System
-    # A form object used to update organizations from the system dashboard.
-    #
-    class UpdateOrganizationForm < Form
-      include TranslatableAttributes
-      include JsonbAttributes
+    class UpdateOrganizationForm < BaseOrganizationForm
```

Comment:
Let's leave it to be consistent with `RegisterOrganizationForm`.
```suggestion
    # A form object used to update organizations from the system dashboard.
    #
    class UpdateOrganizationForm < BaseOrganizationForm
```